### PR TITLE
Add 2019 SunPy projects

### PR DESCRIPTION
**Pull request by nabobalis**
Rough drafts with unknown/unasked mentors.

These projects may be beyond our scope this year.

### CODE REVIEW

File: _projects/2019/sunpy/mag_field_draft.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+requirements:
+ - Knowledge of vector calculus and differential equations
+ - Familiar with numerical methods
+ - Some familiarity with electromagnetism, MHD, potential/force-free magnetic fields (optional)
+difficulty: intermediate
+issues:
+ - https://groups.google.com/forum/#!topic/sunpy/ZLu_HoX_lbc
+ - https://github.com/sunpy/solarbextrapolation/issues/12
+mentors:
+ - dstansby
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply _projects/2019/sunpy/mag_field_draft.md]: Done.

File: _projects/2019/sunpy/roi_draft.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+# add a description of your project
+requirements:
+ - Familiarity with Python
+ - Understanding of design patterns
+difficulty: High
+issues:
+ - https://github.com/sunpy/sunpy/issues/164
+ - https://github.com/sunpy/sunkit-image/issues/2
+mentors:
+ - FIND SOMEONE MAYBE
```
Comment (Cadair) [inline _projects/2019/sunpy/roi_draft.md]: @dpshelio ?

File: _projects/2019/sunpy/mag_field_draft.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+requirements:
+ - Knowledge of vector calculus and differential equations
+ - Familiar with numerical methods
+ - Some familiarity with electromagnetism, MHD, potential/force-free magnetic fields (optional)
+difficulty: intermediate
+issues:
+ - https://groups.google.com/forum/#!topic/sunpy/ZLu_HoX_lbc
+ - https://github.com/sunpy/solarbextrapolation/issues/12
+mentors:
+ - FIND SOMEONE MAYBE
```
Comment (Cadair) [inline _projects/2019/sunpy/mag_field_draft.md]: Given @dstansby and @wtbarnes are in the Thesis pit, I think we might have to pass on this one this year, unless someone steps up to mentor it. @Alex-Ian-Hamilton ?

File: _projects/2019/sunpy/helioviewer_draft.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+requirements:
+ - Familiarity with Python
+ - Some familiarity with Unirest/HTTP requests (ideally)
+difficulty: medium
+issues:
+ - https://github.com/sunpy/sunpy/issues/2860
+ - https://github.com/sunpy/sunpy/issues/2762
+ - https://github.com/sunpy/sunpy/issues/2878
+mentors:
+ - FIND SOMEONE MAYBE
```
Comment (Cadair) [inline _projects/2019/sunpy/helioviewer_draft.md]: I would be ok to mentor this one, @wafels would the helioviewer developer be able to help on this one?

**Comment by DanRyanIrish** [comment]
I'm happy with the ndcube one.

**Comment by wafels** [comment]
yes i will mentor sunkit image

File: _projects/2019/sunpy/mag_field_draft.md (hunk 3 above)
Comment (nabobalis) [reply _projects/2019/sunpy/mag_field_draft.md]: Probably best to drop it for this year.

Comment (Alex-Ian-Hamilton) [reply _projects/2019/sunpy/mag_field_draft.md]: I'd be willing to mentor this.

Comment (Cadair) [reply _projects/2019/sunpy/mag_field_draft.md]: Put me down as a second :roll_eyes:

**Comment by wafels** [comment]
I'll ask him.  Also, put me down too.
I have another project I'll upload tonight.

**Comment by nabobalis** [comment]
Probably should be squash merged at the end.

**Comment by bsipocz** [comment]
> Probably should be squash merged at the end.

it doesn't matter here, ship it instead as soon as possible.

**Comment by nabobalis** [comment]
We didn't finish the `roi_draft.md`

**Pull request merged**